tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -562,8 +562,13 @@ export class WebGPUBackend extends KernelBackend {
     info.bufferInfo.buffer = this.acquireBuffer(info.bufferInfo.byteSize);
 
     if (info.values) {
+      let gpuValues = info.values;
```

Comment:
Could we remove `gpuValues` here?  Could we directly use `info.values = Float32Array.from(info.values);` below?

Reply:
For most case, it is Float32Array, so no need to do this for Float32Array.